src/main/webapp/i18n/en/global.json

```diff
@@ -166,7 +166,8 @@
     "idexists": "A new {{ entityName }} cannot already have an ID",
     "noname": "A new {{ entityName }} cannot be created without a name",
     "idnull": "Invalid ID",
-    "noResultFound": "No result found"
```

Comment:
will this remove impact other pages?
just want to confirm

Reply:
Nice catch here. Sorry that I didn't check before making the change.

Reply:
Resolved in fcd046ac686a64140f789018c6a348f9f36a6866